clang/include/clang/Driver/Options.td

```diff
@@ -7300,8 +7300,9 @@ defm sycl_allow_device_image_dependencies: BoolOptionWithoutMarshalling<"f", "sy
 def fsycl_dump_device_code_EQ : Joined<["-"], "fsycl-dump-device-code=">,
   Flags<[NoXarchOption]>,
   HelpText<"Dump device code into the user provided directory.">;
-def fsyclbin : Flag<["-"], "fsyclbin">,
-  HelpText<"Create a SYCLBIN file">;
+def fsyclbin_EQ : Joined<["-"], "fsyclbin=">,
+  HelpText<"Output in the SYCLBIN binary format in the state specified by <arg> (input, object or executable (default))">;
+def fsyclbin : Flag<["-"], "fsyclbin">, Alias<fsyclbin_EQ>;
```

Comment:
```suggestion
def fsyclbin_EQ : Joined<["-"], "fsyclbin=">,
  Values<"executable,object,input">,
  HelpText<"Output in the SYCLBIN binary format in the state specified by <arg> (input, object or executable (default))">;
def fsyclbin : Flag<["-"], "fsyclbin">, Alias<fsyclbin_EQ>, AliasArgs<["executable"]>;
```
Some suggested updates here.  Use of `AliasArgs` allows for the default to be set without needed to check for 'empty' later on.